06-html5-apis/00_Storage_DnD_WebSockets/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>HTML APIs</title>	
+	<link rel="stylesheet" type="text/css" href="css/index.css">
+	<script type="text/javascript" src="js/index.js"></script>
+</head>
+
+<body>
+	<div class="container">
+		<section>
+		<fieldset>
+			<h1> Insert a new Player here! </h1>
+			<label for="name"> Player Name:</label><br>
+			<input id="playerName" type="text" name="name" placeholder="Insert player name"></input>
+			<br><br>
```

Comment:
Just FYI, br's shouldn't be used for styling purpose, only to provoke a carriage-return. If you want to space things up, try to do through css :)